src/ray/object_manager/plasma/store.cc

```diff
@@ -345,9 +347,10 @@ Status PlasmaStore::ProcessMessage(const std::shared_ptr<Client> &client,
     const auto &request = flatbuffers::GetRoot<fb::PlasmaCreateRequest>(input);
     const size_t object_size = request->data_size() + request->metadata_size();
 
-    auto handle_create = [this, client, message](bool fallback_allocator,
-                                                 PlasmaObject *result,
-                                                 bool *spilling_required) {
+    // absl failed analyze mutex safety for lambda
```

Comment:
Can we do something like add an assert held statement for the lock? The might satisfy absl

Reply:
Is absl smart enough that you can enable analysis with the assert in?

Reply:
:( sadly no